intro_to_python/intro_to_python.md

```diff
@@ -242,7 +280,7 @@ Once you've downloaded Anaconda, you can open the Anaconda Navigator; when you d
 ![Anaconda Navigator interface.](media/anaconda.png?raw=true)
 </div>
 
```

Comment:
Include a reminder and link here to Google Colab since it is mentioned as an option in the quiz?

Reply:
Done! Added it in its own new subsection.